### PR TITLE
doc: device_mgmt: Fix typo in link text

### PR DESCRIPTION
**Pull request by lluiscampos**
Previously it read "there is a sample sample that..."

### CODE REVIEW

**Review by nordicjm — APPROVED** [review]

**Review by de-nordic — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Comment by lluiscampos** [comment]
> To celebrate this milestone and showcase your contribution, we'd love to award you the Zephyr Technical Contributor badge. If you're interested, please claim your badge by filling out this form: [Claim Your Zephyr Badge](https://forms.gle/oCw9iAPLhUsHTapc8).

Thank you. But I'd rather wait until I get a proper technical contribution merged :blush:

**Pull request merged**
